### ISSUE TITLE
[a11y]: Pagination - Jaws SR is not reading Next button

### ISSUE THREAD

**Issue by tay1orjones** [carbon-design-system/carbon#14681]
[Original report](https://ibm-studios.slack.com/archives/C054WK4ES5S/p1694709381258889)

> Jaws SR is not reading Next button when our focus moves from Page Number dropdown to Next Button
>
> Jaws SR is not reading Next button when our focus moves from Page Number dropdown to Next Button while pressing TAB key but when we do shift TAB its working fine. We validated using NVDA SR where it is reading as NEXT button. May be its User Agent but (Jaws bug).

**Comment by andreancardona** [comment]
I'm not seeing the issue 👀 
https://github.com/carbon-design-system/carbon/assets/32720851/450e2f27-baa8-4a5c-8e05-83217612564d

**Comment by andreancardona** [comment]
I'm not seeing the issue 👀 
https://github.com/carbon-design-system/carbon/assets/32720851/52168186-b775-420e-8833-15c5040792b9
